nav2_mission_executor/README.md

```diff
@@ -0,0 +1,82 @@
+# Mission Executor
+
+The Mission Executor module is a task server that coordinates other tasks. It can be used to direct the activities multiple robots as well as other, non-robot tasks.
+
+## Overview
+ 
+The Mission Executor task server receives a **MissionPlan** and returns whether the plan was successfully executed, failed, or was canceled.
+
+```C++
+namespace nav2_tasks {
+
+using ExecuteMissionCommand = nav2_msgs::msg::MissionPlan;
+using ExecuteMissionResult = std_msgs::msg::Empty;
+
+using ExecuteMissionTaskClient = TaskClient<ExecuteMissionCommand, ExecuteMissionResult>;
+using ExecuteMissionTaskServer = TaskServer<ExecuteMissionCommand, ExecuteMissionResult>;
+
+} //  namespace nav2_tasks
+```
+
+NOTE: An Empty message is used in the definition of the task client and server if there is no other data required.
+
+The [mission plan message](https://github.com/ros-planning/navigation2/blob/master/nav2_msgs/msg/MissionPlan.msg) consists of a header and a mission plan string:
+
+```
+std_msgs/Header header
+string mission_plan
+```
+
+The mission plan itself is an XML string that defines the behavior tree. For example, the following XML string specifies a sequence of three NavigateToPose actions:
+
+```XML
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <SequenceStar name="root">
+      <NavigateToPoseAction position="10;11;0.0" orientation="0.7071;0;0.7071;0"/>
```

Comment:
Is there a reason XML was chosen here? Much of the community is in yaml-land and I'd hate to have to support multiple formats